wp-content/themes/access/views/components/alert.twig

```diff
@@ -1,11 +1,20 @@
-<article class="c-alert-box color-{{ this.alert_color }}-status color-mid-background {{ class }}" role="alert">
+<article {% if this.item_scope %}itemscope itemprop="{{ this.item_scope }}" itemtype="https://schema.org/{{ this.item_scope }}" {% endif %}class="c-alert-box color-{{ this.status.type }}-status color-mid-background {{ class }}" role="alert">
+  {% if this.item_scope %}
+    <meta itemprop="name" content="{{ this.post_title }}" />
+    <meta itemprop="datePosted" content="{{ this.post_date }}" />
+  {% endif %}
+
   <div class="c-alert-box__icon">
     <svg aria-hidden="true" class="c-alert-box__svg icon icon-{{ this.alert_icon }}">
       <use xlink:href="#icon-{{ this.alert_icon }}"></use>
     </svg>
   </div>
 
-  <div class="c-alert-box__body">
-    {{ this.alert_content }}
+  <div {% if this.itemprop_body %}itemprop="{{ this.itemprop_body }}" {% endif%}class="c-alert-box__body">
+      {{ this.alert_content }}
   </div>
+
+  {% if this.item_scope %}
+    <meta itemprop="category" content="https://www.wikidata.org/wiki/Q81068910" />
```

Comment:
This category is specific to COVID, so it should also only be present if the item scope status also equals `covid-response`. this may be best done in the controller.

Reply:
@devowhippit does alert have fallback schema?
`this.item_scope` will either return `false` or `SpecialAnnouncement`, we can improve readability by `{% if this.item_scope  ==  SpecialAnnouncement %}` or `{% this.status['type'] == 'covid-response'  %}` Let me know if you agree with this implementation or if we need to add a fallback  schema.

Reply:
@theasteve I am not certain if there is an applicable fallback. There may be something on this list but I think we can scope the task to just the SpecialAnnouncement schema https://schema.org/docs/full.html I think your readability suggestion is good!